### DISCUSSION TITLE
Event of switching to night mode and vice versa

### DISCUSSION THREAD

**Discussion by de-salvatierra** [fwestenberg/reolink_dev#225] [Q&A]
is it possible to get the Event of switching to night mode and vice versa?

**Comment by fwestenberg** [comment]
I suggest to use an automation with state trigger. The camera entity has all attributes corresponding to the services, so this:

![image](https://user-images.githubusercontent.com/47930023/126440601-07e34a25-5ac5-4c87-8541-d981717fa0f8.png)

should work.
